### PR TITLE
PERF-85: Modified JMX to be compatible with Goldenrod

### PR DESCRIPTION
**Pull request by mtraneis**
- Renamed test to "circulation_checkInCheckOut"
- Moved the HTTP Request steps into either "Check-Out" or "Check-In" Controller.  Used the Controllers to automatically tally up total response time of all requests in Check-out or Check-in transactions
- Added a suffix "(Check out)" or ("Check in") to the end of the HTTP Request names to easily identify in JMeter output that the request belongs to check out or check in.
- Updated the query strings of the HTTP Requests, removed or added new ones according to a temporary JMX captured by Blazemeter.

### CODE REVIEW

File: workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <collectionProp name="Arguments.arguments">
           <elementProp name="HOSTNAME" elementType="Argument">
             <stringProp name="Argument.name">HOSTNAME</stringProp>
-            <stringProp name="Argument.value">${__P(HOSTNAME,[okapi-hostname-without-http-protocol])}</stringProp>
+            <stringProp name="Argument.value">${__P(HOSTNAME,okapi-fcp1-us-east-1.int.aws.folio.org)}</stringProp>
```
Comment (javalkar-varun) [inline workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx:18]: We should not check-in hardcoded okapi url?

**Review by javalkar-varun — APPROVED** [review]
I did not verify by running a test with this JMX file but I think you already checked it. Other than one comment, looks good to me.

File: workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx (hunk 1 above)
Comment (mtraneis) [reply workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx:18]: I'll fix it, thanks.

**Comment by mtraneis** [comment]
> I did not verify by running a test with this JMX file but I think you already checked it. Other than one comment, looks good to me.

Yeah, I have been testing with this JMX but I haven't created a PR for it until now.  I will need to modify it again for Honeysuckle, though.

**Pull request merged**